config/subresponder_schema.toml

```diff
@@ -0,0 +1,99 @@
+[reference]
+Description = "nan"
+Deduced_Data_Type = "int64"
+Nullable = false
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = 11001603625
+Max_values = 19891309165
+Possible_Categorical_Values = ["nan"]
+
+[period]
+Description = "nan"
+Deduced_Data_Type = "int64"
+Nullable = false
+Current_Data_Type = "str"
+Length = 6
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = [202012]
+
+[survey]
+Description = "All values are 002"
+Deduced_Data_Type = "str"
+Nullable = false
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["002"]
+
+[questioncode]
+Description = "nan"
+Deduced_Data_Type = "int64"
+Nullable = false
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = "101"
+Max_values = "714"
+Possible_Categorical_Values = ["nan"]
+
+[response]
+Description = "nan"
+Deduced_Data_Type = "int64"
```

Comment:
2023-08-01 11:30:55,274 - src.staging.validation - validate_data_with_schema - DEBUG:response before: object
2023-08-01 11:30:55,274 - src.staging.validation - validate_data_with_schema - ERROR:invalid literal for int() with base 10: '2.2'

This should be an object/string since it contains everything